### DISCUSSION TITLE
improve chia plots check

### DISCUSSION THREAD

**Discussion by Hardcore-fs** [Chia-Network/chia-blockchain#6839] [Ideas]
1.when validating the plots, also include a "number of number" for each plot
so you know which plot you are on.

2. at the end " WARNING n plots invalid" ,is not real helpful......  print the  plot file names.

**Comment by gromran** [comment]
Where is the problem? Parse the entire log for "Error:  ..."

**Reply by Hardcore-fs** [reply]
it is not an error it is in discussions.
there needs to be more info in the log.

when checking plots it just says there is an "error", if you have to check  thousands of plots,
 it is just extra human work that a computer can do for free..
 by storing the file of the bad plot file name & printing it at the end.

same with showing the number of the current plot being processed out of how many plots were found to be processed.

**Reply by gromran** [reply]
> when checking plots it just says there is an "error", if you have to check thousands of plots,

you can parse the entire log after checking with an simple, short script (bash, python, whatever)

**Reply by Hardcore-fs** [reply]
yep.. I hardy need anyone to explain what we can and cannot do programming wise to parse output of log  files.

What you mean  is we have to do extra work the computer can do for free....
when all it takes is a few python code lines to:
1. add a loop/fixed count to the printout
2.  open an array, stick the file name of the defective file in it
3. iterate the array at the end of the process.

**Comment by specialmans** [comment]
我们check的主要目的是为了找出错误的图，而不是错误的数量，建议check后能print出错误的plot

**Comment by Loveatslick** [comment]
[https://user-images.githubusercontent.com/86172357/122761569-3d720a80-d251-11eb-8949-b988143d370b.MOV]()

**Reply by Loveatslick** [reply]
- - - - - [ ] https://user-images.githubusercontent.com/86172357/122761743-6b574f00-d251-11eb-8095-12f29b225d22.MOV[********##^#=¥>¥********]()

**Reply by Loveatslick** [reply]
> - - - - - [ ] https://user-images.githubusercontent.com/86172357/122761743-6b574f00-d251-11eb-8095-12f29b225d22.MOV[********##^#=¥>¥********]()

🔥👚🆔📶🎦▶️🆙⏏️🔄🔀⤴️
